submodules/eks/k8s.tf

```diff
@@ -22,8 +22,8 @@ resource "terraform_data" "run_k8s_pre_setup" {
   ]
 
   provisioner "local-exec" {
-    command     = "./${module.k8s_setup[0].filename} set_k8s_auth set_eniconfig"
-    interpreter = ["bash", "-c"]
+    command     = "${module.k8s_setup[0].filename} set_k8s_auth set_eniconfig"
+    interpreter = ["bash"]
```

Comment:
The command is a single string no matter what, so you might need to do something like this:

```
    command     = "bash ${module.k8s_setup[0].filename} set_k8s_auth set_eniconfig"
    interpreter = ["bash", "-c"]
```

Or else this, even though it's weird:

```
    command     = "set_eniconfig"
    interpreter = ["bash", module.k8s_setup[0].filename, "set_k8s_auth"]
```

Interpreter is optional, so long as it's equivalent to `bash -c` you might be able to just omit it too.

Reply:
yea, going through all the stupid options... garbage.... could just make it all interpreter too.